Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -1175,6 +1195,8 @@ static void UpdateBATs(BatTable& bat_table, u32 base_spr)
           valid_bit = 0x3;
         else if ((address >> 28) == 0xE && (address < (0xE0000000 + Memory::L1_CACHE_SIZE)))
           valid_bit = 0x3;
+        if (overlaps_memcheck(address | ((1 << BAT_INDEX_SHIFT) - 1)))
```

Comment:
I actually was supposed to use this expression as the memcheck addresses are in virtual